app/javascript/hyacinth_v1/components/batch_imports/digital_object_imports/DigitalObjectImportIndex.js

```diff
@@ -59,8 +59,10 @@ function DigitalObjectImportIndex() {
   };
 
   const onTabClick = (status) => {
+    // ALL CAPS enum value per graphql API
+    const statusUcase = status.toUpperCase();
```

Comment:
Would it be possible to omit this `toUpperCase` here if we change the `<StatusNavItem>` components in the same file to use capitalized eventKey values?  (I haven't done a deep dive into the code, so it might not be that straightforward.)